joern-cli/build.sbt

```diff
@@ -46,7 +46,8 @@ lazy val downloadFuzzyPreprocessor = taskKey[File]("Download the FuzzyC2CPG prep
 downloadFuzzyPreprocessor := {
   val ppFilename = "fuzzyppcli.zip"
   val ppUrl = new URL(
-    s"https://github.com/ShiftLeftSecurity/fuzzyc2cpg/releases/download/v${Versions.fuzzyc2cpgVersion}/$ppFilename")
+    // s"https://github.com/ShiftLeftSecurity/fuzzyc2cpg/releases/download/v${Versions.fuzzyc2cpgVersion}/$ppFilename")
+      "https://github.com/ShiftLeftSecurity/fuzzyc2cpg/releases/download/v1.1.17/fuzzyppcli.zip")
```

Comment:
Did you mean to leave this commented out?

Reply:
that was accidental, thanks for spotting!